### ISSUE TITLE
add a one-click-apps Category

### ISSUE THREAD

**Issue by guilh22** [caprover/caprover#2173]
**Is your feature request related to a problem? Please describe.**
No it is not related to a problem. The app catalog start to become huge and even if we have a search field, I find it would be awesome to have all app ordered into Categories and that we could add those category to the catalog page as filter in the webapp.

**Describe the solution you'd like**
- Add a category (could be one or multiple) to all our app in the one-click-app
- Gathered a list of all category created
- Add all those category as filter in the one-click-app catalog of the webapp

**Describe alternatives you've considered**
Like I said, we can still use the search field to find specific container, but this is not very useful for non-technical person.

**Additional context**
Here is a quick example![image](https://github.com/user-attachments/assets/c99c087a-3fcd-4a89-9531-b0b14eb10e90)

**Comment by githubsaturn** [comment]
Thanks for the suggestion! I think a more generic approach would be "tag" system as it'll allow each one click app to have multiple tags. e.g., one app could be used both for logging and alerting.

**Comment by guilh22** [comment]
Yep! That's what I meant by `Add a category (could be one or multiple)`. It would be a really nice addition to the tool and allow more user to use caprover as it allow you to discover new application per tags

**Comment by margani** [comment]
Can we add a sort feature like the one explained in #1098?
